monolith/views/users.py

```diff
@@ -37,6 +37,10 @@ def create_user():
                 type_message=REGISTRATION_EMAIL,
                 params=[user.email, user.lastname, "112344"],
             )
+            newrole = db.session.query(Role).filter_by(id=user.role_id).first()
```

Comment:
This code give problem inside the tests with 401! 

I lost the logic inside the code but at the moment we have a clean db in testing phases.

Another good change to do is move the name from `newrole` to  `new_role`

Reply:
I don't understand why it would. And it's needed if we want to auto-login the user after the registration. We could get rid of it forcing the user to login manually after the registration. Otherwise, not setting the session var, the navbar and maybe other methods that use this variable to determine the role of the user cannot work properly until the user Logout and login manually

Reply:
I think that the problem is related to @roles_allowed(roles=["HEALTH"]), or better, about this annotation find the role